aten/src/ATen/native/quantized/cpu/qconv_unpack.cpp

```diff
@@ -55,9 +57,10 @@ class QConvUnpackWeightsInt8 final : public c10::OperatorKernel {
       unpacked_weights = _empty_per_channel_affine_quantized_like(
           scales.toType(kDouble),
           zero_points.toType(kLong),
-          {output_channels, kernel_h, kernel_w, C_per_G},
+          {output_channels, C_per_G, kernel_h, kernel_w},
```

Comment:
Do we need to change it similarly for QNNPACK as well below since we store it as `MemoryFormat::ChannelsLast` in the packed struct?

Reply:
No, you store the original Tensor and it's fine to return Tensor in ChannelLast format (it'd just happen to be non-contiguous but still semantically correct)